tests/utils.go

```diff
@@ -355,9 +355,14 @@ func (w *ObjectEventWatcher) Watch(abortChan chan struct{}, processFunc ProcessF
 	go func() {
 		defer GinkgoRecover()
 		for obj := range eventWatcher.ResultChan() {
-			if f(obj.Object.(*k8sv1.Event)) {
-				close(done)
-				break
+			switch t := obj.Object.(type) {
+			case *k8sv1.Event:
+				if f(obj.Object.(*k8sv1.Event)) {
+					close(done)
+					break
+				}
+			default:
+				fmt.Printf("WARNING: unexpected event type %T", t)
```

Comment:
Can you please log the content? Also fail the test? We should try to understand what this is.